docs/reference/cli.md

```diff
@@ -726,14 +730,19 @@ View a lineage record.
 $ nextflow lineage view <lid>
 ```
 
+:::{tip}
+You can use the [jq](https://jqlang.org/) command-line tool to apply further queries and transformations on the resulting lineage record.
+:::
+
 The output of a workflow run can be shown by appending `#output` to the workflow run LID:
 
 ```console
 $ nextflow lineage view lid://c914d714877cc5c882c55a5428b510b1#output
 ```
 
-:::{tip}
-You can use the [jq](https://jqlang.org/) command-line tool to apply further queries and transformations on the resulting lineage record.
+:::{warning}
+Only outputs described using the {ref}`workflow-outputs` syntax are captured in lineage records.
+If your pipeline only uses {ref}`publishDir` then an empty list will be returned.
```

Comment:
@jorgee this would be a good place to explain how files published via `publishDir` can still be accessed as LIDs

Reply:
Great. I am going to add it.